charts/argo-cd/templates/argocd-server/certificate.yaml

```diff
@@ -14,11 +14,11 @@ spec:
   commonName: {{ .Values.server.certificate.domain | quote }}
   dnsNames:
     - {{ .Values.server.certificate.domain | quote }}
-  {{- range .Values.ingress.additionalHosts }}
+  {{- range .Values.server.ingress.additionalHosts }}
```

Comment:
Should this be `server.certificate.additionalHosts` as I do not see this value under `server.ingress`